### ISSUE TITLE
Read audio file from a MemoryStream ?

### ISSUE THREAD

**Issue by ajonno** [Zeugma440/atldotnet#149]
hi, is it possible to create a Track object by using a MemoryStream ? (instead of a physical file location)

**Comment by Zeugma440** [comment]
Sure. Did you take a closer look at the constructors ?

**Comment by ajonno** [comment]
yes i saw there was a "stream" ctor option but I wasnt sure how to define P3.

```Track theTrack = new Track(ms, mimeType: "audio/mp3", null);``` <=== p3 here

Also is a MemoryStream a valid object for P1 ? Or does it need to be a (say) FileStream ?

**Comment by ajonno** [comment]
Pretty sure I tried the API with a MemoryStream but it didn't work. I'll double check that today
